src/Dscp/DB/DSL/Interpret/Sqlite3/Schema.hs

```diff
@@ -0,0 +1,186 @@
+
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.DB.DSL.Interpret.Sqlite3.Schema (schema) where
+
+import Universum
+
+import Text.InterpolatedString.Perl6 (q)
+
+schema :: IsString s => s
+schema = [q|
+
```

Comment:
Wait, what kind of copypaste is that? You already have `database/schema.sql`, just use `quoteFile` here: http://hackage.haskell.org/package/template-haskell-2.7.0.0/docs/Language-Haskell-TH-Quote.html

Reply:
They left spaces inside quasi-quoter in that version of documentation. It took me time to figure out, that spaces do matter there, and they fixed it in latest version (`2.13.0.0`) of docs. My bad, I didn't checked if docs are fresh enough.